udata/core/topic/views.py

```diff
@@ -44,6 +44,7 @@ def display(topic):
     return theme.render(
         'topic/display.html',
         topic=topic,
+        datasets=[d for d in topic.datasets if hasattr(d, 'pk')],
```

Comment:
If we're running the `purge_datasets` command regularly, is that change still useful?